build_tools/scripts/bazel_to_cmake_targets.py

```diff
@@ -36,9 +36,14 @@ def _convert_absl_target(target):
     target_name = target.rsplit("/")[-1]
   return "absl::" + target_name
 
+DEAR_IMGUI_EXPLICIT_TARGET_MAPPING = {
+  "@dear_imgui": "dear_imgui::dear_imgui",
+  "@dear_imgui//:imgui_sdl_vulkan": "dear_imgui::impl_sdl\n    dear_imgui::impl_vulkan",
```

Comment:
Can we instead change these mappings to be one to many so we don't have to do weird string manipulation here? Map targets would return a list which the caller would be responsible for formatting properly. Could be a follow-up

Reply:
I like the idea, but I wasn't successful with a one liner that transforms only sublists. So would prefer a follow-up.

Reply:
Sounds good. I think I would just make all the convert_target methods return lists